python/cudf/dataframe/dataframe.py

```diff
@@ -970,8 +987,36 @@ def _sanitize_values(self, series, SCALAR):
         """
         if SCALAR:
             col = series
-        if not isinstance(series, Series):
+
+        if (not isinstance(series, Series) and utils.is_list_like(series)) \
```

Comment:
`utils.is_list_like(series)` will always return false if the input dtype is a `Series` so you can eliminate this first condition entirely.

Reply:
eliminated it